specification/implementation/metadata/mets/amdsec/index.md

```diff
@@ -1,10 +1,17 @@
 ### 5.3.4.	Use of the METS administrative metadata section (element amdSec)
-The purpose of the METS administrative data section is to embed or refer to files containing administrative metadata about the IP content objects. CSIP is only using referencing of files containing administrative metadata. The CSIP (and METS) categorises preservation metadata as administrative metadata, specifically Digital Provenance metadata (following the avaiable guidelines), hence all preservation metadata should be referenced from a digiprovMD element within the amdSec.
+METS defines an administrative metadata section, used to embed metadata or provide references to files containing metadata for the information package content. The CSIP encourages the use of references to metadata files, rather than embedding metadata in the `amdSec` element, and the examples reflect this.
 
-The administrative metadata section also contains rights statements. Regardless of used standard used to record the rights metadata it should be referenced from a rightsMD element within the amdSec. Suitable standards include the collaborative work from RIGHTSTATMENTS.ORG <http://rightsstatements.org> , Europena right statements <https://pro.europeana.eu/page/available-rights-statements> , the METS Rights Schema <https://github.com/mets/METS-Rights-Schema> created and maintained by the METS Board, the rights sections of PREMIS <http://www.loc.gov/standards/premis/> as well as any local rights statements.
```

Comment:
Sorry final point, do we mean to remove the extra information about rights schema?

Reply:
I would say no. But we also have it in the profile. So the question is where it should be. In the table or in the text?